portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/displaypagetemplate/DisplayPageTemplates.testcase

```diff
@@ -77,6 +77,43 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-129360. Change the Portlet Friendly URLs in Display Page Templates to list display style."
```

Comment:
The user could change the display style of Documents and Media in display page template.